jest.config.js

```diff
@@ -2,6 +2,6 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-const config = require('@polkadot/dev/config/jest');
-
-module.exports = Object.assign({}, config);
\ No newline at end of file
+module.exports = {
```

Comment:
The reason tests were failing for me, was that polkadot/dev is using babel, and here we're using tsc. ts-jest is a tsc wrapper around jest

Reply:
We could switch to babel (main argument: faster compile times), but tsc is the official one and imo simpler to use

Reply:
let's stick with tsc